HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,141 @@
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+ggplot(diamonds, aes(cut, color = clarity, fill = clarity)) + #plot cut, color/fill based on clarity
+  geom_bar(position = "dodge") + #write bars, dodged
+  labs(x = "Diamond Cut", y = "Number of Diamonds", title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by\ntype of cut quality and clarity of diamond") + #add all labels
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") + #add caption
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.4) #add highlight rect
+  
+
+
+
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+iris$SpeciesOrdered <- factor(iris$Species, levels = c("versicolor", "setosa","virginica")) #good order
+
+iris_plot <- ggplot(iris, aes(Sepal.Length, Petal.Length,color = Species, shape = Species))
+
+iris_plot + facet_wrap(vars(SpeciesOrdered),scales = "free") + #make three subplots
+  scale_color_manual(values = c("green","red","dodgerblue2")) + #specify colors
+  scale_shape_manual(values = c(17, 16, 15)) + #specify shapes
+  scale_x_continuous(limits = c(4.2, 8)) + #set xlim
+  geom_point() + #plot points
+  geom_smooth(method = "lm", se = F, color = "black") #plot trend line
+
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graoh 3 code}
+
+ggplot(corvette, aes(displ,hwy,label = paste("Corvette,",year))) + #plot corvette data
+  labs(title = "Corvettes are a bit of an outlier") + #add title
+  geom_point(color = "blue") + #corvettes are blue
+  geom_text_repel() + #add corvette text
+  geom_point(data = mpg[mpg$model != "corvette",], color = "black") + #plot all other data
```

Comment:
I love how versatile R is, you coded this totally different (and almost oppositely) than I did and yet we still have almost identical graphs! So cool. 
I went from the perspective of keeping my ggplot() command blank and then specifying geom_point() commands for each the whole data set and the corvette subset just in case you were curious :)